### PR TITLE
Fixed eddiebot broken readme link

### LINKED ISSUE #3022: [BUG] Broken link in support for Open source tips from EddieBot
### Description of the bug

The link in the support readme for Open source tips from EddieBot was broken

### Project stack or language (e.g. front-end)

readme file

### Working Environment (e.g. operating system, browser, device)

browser

### Please add screenshots (if applicable)

pr2

### Add any other context about the problem here

_No response_

### PR DESCRIPTION
**Pull request by pranv-sonawane**
closes #3022

#### What does this PR do?
this fixes the broken link found in the support readme section for Open source tips from EddieBot
#### Description of the task to be completed?
the link has been modified

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
 
   Here is a list of resources that also might help you start contributing to open source:
   - [How to get started in hackathons](https://github.com/EddieHubCommunity/support/issues/32)
-  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/tips)
-- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](./tips/finding-open-source-projects.md)
+  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/docs/tips)
+- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](https://github.com/EddieHubCommunity/support/blob/main/docs/tips/finding-open-source-projects.md)
```
Comment (schmelto) [inline README.md:74]:
```suggestion
- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](./docs/tips/finding-open-source-projects.md)
```

**Comment by schmelto** [comment]
Could you add the issue number in the pr description with for example `resolve #123 `

File: README.md (hunk 1 above)
Comment (schmelto) [reply README.md:74]: I suggest using this annotation

**Review by krishguptadev — DISMISSED** [review]
Just @schmelto's suggestions and then 🚀

File: README.md (hunk 1 above)
Comment (pranv-sonawane) [reply README.md]: hi @schmelto i did another pr for that

File: README.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
 
   Here is a list of resources that also might help you start contributing to open source:
   - [How to get started in hackathons](https://github.com/EddieHubCommunity/support/issues/32)
-  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/tips)
-- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](./tips/finding-open-source-projects.md)
+  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/docs/tips)
```
Comment (schmelto) [inline README.md:73]:
```suggestion
  - [Open source tips from EddieBot](./docs/tips)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Review by schmelto — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Am I missing something? This appears to be a duplicate of #3021?

**Comment by krishguptadev** [comment]
Yes its a duplicate 🤔

**Comment by nhcarrigan** [comment]
Hey there,

Thanks for your interest in contributing. However this appears to be a duplicate of your previous pull request - as such, we are unable to accept these changes.

We encourage you to continue contributing by looking for issues labelled `help wanted`!

**Pull request closed without merging**